### PR TITLE
TE-674 Add pinterest icon

### PR DESCRIPTION
**Pull request by Jonnygwi**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-674)

### What **one** thing does this PR do?
Adds Pinterest icon

### Any other notes
Change the name of the `google` Icon to `google plus`

image

### CODE REVIEW

File: src/components/elements/Icon/Readme.md [hunk 1]
```diff
@@ -118,10 +118,12 @@ Library icons
 <Heading size="small">Social media</Heading>
 <Icon labelText="facebook" name="facebook" />
 <Divider />
-<Icon labelText="google" name="google" />
+<Icon labelText="google plus" name="google plus" />
```
Comment (stephenkr) [inline src/components/elements/Icon/Readme.md]: I think it should be `googleplus`

File: src/components/elements/Icon/component.js [hunk 2]
```diff
@@ -114,7 +114,7 @@ Component.propTypes = {
     'double bed',
     'facebook',
     'fire',
-    'google',
+    'google plus',
```
Comment (stephenkr) [inline src/components/elements/Icon/component.js]: Same as above. I think it should be `googleplus`

File: src/components/elements/Icon/Readme.md (hunk 1 above)
Comment (marcdavi-es) [reply src/components/elements/Icon/Readme.md]: In future - let's not lose time on these kinds of discussions. Fall back on what Semantic UI does... https://react.semantic-ui.com/elements/icon

In this case... `google plus`

Comment (stephenkr) [reply src/components/elements/Icon/Readme.md]: Ok, sounds good.

**Pull request merged**